README.md

```diff
@@ -70,6 +70,42 @@ new RollbarConfig
 </dt>
 <dd>A string URI which will be used as the WebClient proxy by passing to the WebProxy constructor.
 </dd>
+<dt>CheckIgnore
+</dt>
+<dd>Function called before sending payload to Rollbar. Return `true` to stop the error from being sent to Rollbar.
+</dd>
+<dt>Truncate
+</dt>
+<dd>Truncates the payload before sending it to Rollbar.
+</dd>
+<dt>Server
+</dt>
+<dd>The server name, as a string.
+</dd>
+<dt>Person
+</dt>
+<dd>You can set the current person data like so
+
+```csharp
+private void SetRollbarReportingUser(string id, string email, string userName)
+       {
+           Person person = new Person(id);
+           person.Email = email;
+           person.UserName = userName;
+           RollbarLocator.RollbarInstance.Config.Person = person;
+       }
+```
+
+and this person will be attached to all future Rollbar calls.
+</dd>
+<dt>MaxReportsPerMinute
+</dt>
+<dd>The maximum reports sent to Rollbar per minute, as an integer.
+</dd>
+<dt>ReportingQueueDepth
+</dt>
+<dd>The reporting queue depth, as an integer. 
```

Comment:
I'd like more information about what exactly this means.

Reply:
The way new notifier works is:
Multiple instances of RollbarLogger can be created. Each instance does not really send payloads when its reporting methods are called. It just creates payload object and drops it into a singleton dictionary of queues that are indexed by unique access_token values. The RollbarQueue controller is in charge of actually sending/retrying/etc. of the payloads across all the queues while obeying reporting rate limits. The queues can be assigned a queue depth to limit reports bursts when connectivity to the Rollbar API server is too poor.  
More details are in the attached file (please, see the "Conceptual Design of the Notifier" section)
[Rollbar.NET Notifier Documentation.docx](https://github.com/rollbar/Rollbar.NET/files/1617780/Rollbar.NET.Notifier.Documentation.docx)